firestore/app/src/main/java/com/google/example/firestore/DocSnippets.java

```diff
@@ -587,23 +587,24 @@ public void onFailure(@NonNull Exception e) {
 
     public void writeBatch() {
         // [START write_batch]
-        // Get a new write batch
-        WriteBatch batch = db.batch();
+        final DocumentReference nycRef = db.collection("cities").document("NYC");
```

Comment:
This API was introduced just to make Kotlin nicer, I don't think we should actually use it in Java.  Personally I think the old one is cleaner.